connectors/sources/github.py

```diff
@@ -711,6 +712,9 @@ async def post(self, query_data, need_headers=False):
             dictionary: Client response
         """
         url = f"{self.github_url}/graphql"
+        self._logger.debug(
+            f"Sending POST to {url} with body: '{json.dumps(query_data)}'"
+        )
```

Comment:
Is there a chance to leak anything dangerous here?

Reply:
Shouldn't be. This won't have credentials - that's in the Authorization header. And it would just have file paths and repo names and file types and such - so identifiers, but not data contents